lib/levelup.js

```diff
@@ -54,7 +55,11 @@ function LevelUP (db, options, callback) {
   this.options = getOptions(options)
   this._db = db
   this.db = new DeferredLevelDOWN(db)
-  this.open(callback)
+  this.open(callback || function(err) {
```

Comment:
```suggestion
  this.open(callback || function (err) {
```

(You can also run` standard --fix` to fix style issues automatically)